ibis/expr/timecontext.py

```diff
@@ -127,6 +127,88 @@ def canonicalize_context(
     return begin, end
 
 
+def construct_time_context_aware_series(
+    series: pd.Series, frame: pd.DataFrame
+) -> pd.Series:
+    """ Construct a Series by adding 'time' in its MultiIndex
+
+    In window execution, the result Series of udf may need
+    to be trimmed by timecontext. In order to do so, 'time'
+    must be added as an index to the Series. We extract
+    time column from the parent Dataframe `frame`.
+    See `trim_with_timecontext` in execution/window.py for
+    trimming implementation.
+
+    Examples:
+        Assume frame is:
+                             time  id  value
+        0 2017-01-02 01:02:03.234   1    1.1
+        1 2017-01-03 01:02:03.234   2    2.2
+        2 2017-01-04 01:02:03.234   3    3.3
+
+        For a series of:
```

Comment:
Can you add some index here for better readability ?

1. For a series:
  ...
   The result series will be

2. For a series:
    ...
    The result will be
    ...

Reply:
Good point, combining with Jeff's comment I update this to be runnable code / ipython output